swift/ios-shoppe-demo/Views/StoreViewController.swift

```diff
@@ -39,6 +37,26 @@ class StoreViewController: UICollectionViewController, UICollectionViewDelegateF
         }
     }
 
+    func setupNavigationBar() {
+        let barCartButton = UIBarButtonItem(image: UIImage(named: "Cart"), style: .done, target: self, action: #selector(openCart))
+        let textAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
+
+        barCartButton.tintColor = .white
+
+        navigationController?.navigationBar.titleTextAttributes = textAttributes
+        navigationController?.navigationBar.barTintColor = .systemIndigo
+
+        navigationItem.title = "Shoppee"
```

Comment:
Typo `Shoppe`.  Patrick and I have these titled as `Reactshoppe` and `Angular Shoppe`.  For consistency, you might consider using `iOS Shoppe`.